test/e2e/main_test.go

```diff
@@ -12,39 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package e2e_test
+package e2e
 
 import (
 	"fmt"
 	"os"
 	"path/filepath"
 	"runtime"
 	"testing"
-
-	e2e "github.com/tetratelabs/getenvoy/test/e2e/util"
 )
 
-// getEnvoy is the absolute path to the "getenvoy" binary used in all tests.
-var getEnvoy = e2e.GetEnvoy
-
 //nolint:golint
 const E2E_GETENVOY_BINARY = "E2E_GETENVOY_BINARY"
 
-// TestMain ensures the "getenvoy" binary is valid.
+// TestMain ensures the "getenvoy" binary is valid and we can read an Envoy reference (version) to run.
```

Comment:
now this change to the comment isn't correct?

Reply:
yep